pyfar/dsp/filter/audiofilter.py

```diff
@@ -232,6 +232,11 @@ def high_shelve_cascade(
 
     ``slope = gain/bandwidth``
 
+    .. note::
+        The `bandwidth` must be at least 1 octave to obtain a good
+        approximation of the desired frequency response. Make sure to specify
+        the parameters `gain`, `slope`, and `bandwidth` accordingly.
```

Comment:
```suggestion
        The ``bandwidth`` must be at least 1 octave to obtain a good
        approximation of the desired frequency response. Make sure to specify
        the parameters ``gain``, ``slope``, and ``bandwidth`` accordingly.
```

Reply:
I considered it to be references for parameters, in which our guide lines suggest italic font.